language-server/modules/langserver-core/src/test/resources/codeaction/create-function/source/createUndefinedFunctionInCompoundStatement3.bal

```diff
@@ -0,0 +1,4 @@
+function test() {
+    string s = "";
+    s += concat(i);
+}
```

Comment:
Shall we add a test case in the module level as well?

Reply:
Added

Reply:
@SasinduDilshara, If we have a function invocation like `mod1:concat(i)`, we do not suggest creating function CA in `mod1`.